<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulator_Widget</class>
 <widget class="QMainWindow" name="Simulator_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="GL_Widget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>62</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_simulation"/>
   <addaction name="action_reset"/>
   <addaction name="action_stop_motion"/>
   <addaction name="separator"/>
   <addaction name="action_move"/>
   <addaction name="action_rotate"/>
   <addaction name="separator"/>
   <addaction name="action_test_ahrs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Status_Widget" name="status"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="Environment_Widget" name="environment"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="Sensors_Widget" name="sensors"/>
  </widget>
  <action name="action_simulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_stop_motion">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Motion</string>
   </property>
   <property name="toolTip">
    <string>Stops the UAV motion</string>
   </property>
  </action>
  <action name="action_move">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/move.png</normaloff>:/icons/move.png</iconset>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="action_rotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/rotate.png</normaloff>:/icons/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="action_test_ahrs">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/test.png</normaloff>:/icons/test.png</iconset>
   </property>
   <property name="text">
    <string>Test AHRS</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/restart.png</normaloff>:/icons/restart.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GL_Widget</class>
   <extends>QWidget</extends>
   <header>GL_Widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Status_Widget</class>
   <extends>QWidget</extends>
   <header>Status_Widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Environment_Widget</class>
   <extends>QWidget</extends>
   <header>Environment_Widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Sensors_Widget</class>
   <extends>QWidget</extends>
   <header>Sensors_Widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
